example/Gruntconfig.json

```diff
@@ -5,8 +5,20 @@
   },
   "domain": "project.vm",
   "packages": {
-    "srcFiles": ["!sites/*/files/**", "!xmlrpc.php", "!modules/php/*"],
-    "projFiles": ["README*", "bin/**"]
+    "default": {
+      "srcFiles": ["!sites/*/files/**", "!xmlrpc.php", "!modules/php/*"],
+      "projFiles": ["README*", "bin/**"]
+    },
+    "acquia": {
+      "srcFiles": ["**", ".**", "!sites/default/**", "!sites/*/files/**", "!??/**", "!sites/all/modules/dev/**", "!sites/*/settings.php", "!sites/*/settings.db.inc", "!sites/*/settings.local.inc", "!.htaccess", "!.gitignore", "!.gitkeep"],
+      "sharedFiles": ["**", "!files/**", "!default.settings.php", "settings.php", "!settings.local.inc"],
+      "projFiles": ["README*", "bin/**"],
+      "root": "build/packages/acquia",
+      "paths": {
+        "docroot": "build/packages/acquia/docroot",
+        "shared": "build/packages/acquia/shared"
+      }
```

Comment:
I followed the setup of the [Vista Campus repo](https://bitbucket.org/phase2tech/vista-campus/), where the `shared` directory is outside of docroot. @jhedstrom set that one up and can probably speak more to the architecture. I think this also speaks to the need to standardize. Since the only requirement for Acquia is that `docroot` is a root folder in the repo, builds might have needs for resources outside of that similar to Vista Campus. Mostly this was an exercise in seeing if this type of segmented structure could work. If we have a preferred repo architecture, we should document it and we can build that in.